Detox.js

```diff
@@ -395,8 +395,8 @@ class Detox extends Helper {
    * I.click({ ios: 'Save', android: 'SAVE' }, '#main'); // different texts on iOS and Android
    * ```
    * 
-   * @param {string|object} locator 
-   * @param {string|object} [context=null] 
+   * @param {CodeceptJS.LocatorOrString} locator 
+   * @param {?CodeceptJS.LocatorOrString} [context=null] 
```

Comment:
What `?` means  in type?

Reply:
`null` or

So basically it is the same as `null | CodeceptJS.LocatorOrString`

Reply:
nice, so, what if add `?` to all functions from this module, which allows `null`?

Reply:
longPress, seeElementExists, dontSeeElementExists

Reply:
I just don't understand, why generated docs does not include `| null` in types
![image](https://user-images.githubusercontent.com/11647141/70305245-fd853700-1814-11ea-8e40-afb96c102bef.png)